src/mpid/ch4/shm/posix/eager/fbox/fbox_types.h

```diff
@@ -19,12 +19,12 @@
 
 typedef struct {
 
-    volatile uint64_t data_ready;
+    volatile uint64_t data_ready;       /*  takes a cache line */
 
     int is_header;
-    size_t payload_sz;
+    unsigned int payload_sz;    /* payload and cell can't be more than 4GB */
 
-    uint8_t payload[MPIDI_POSIX_FBOX_DATA_LEN];
+    uint8_t payload[MPIDI_POSIX_FBOX_DATA_LEN]; /* should be 64-bit aligned */
 
```

Comment:
If you comment "should", then need explain "why", unless it is obvious (is it?). Anyway, there is `MPL_ATTR_ALIGNED` if you want to force/ensure an alignment. The comment can be easily ignored.

Reply:
64-bit may help memcopy. I can change it to use MPL_ATTR_ALIGNED, but it should not be necessary because the fields before payload is already multiple of 64 bits.